share/man/man4/netmap.4

```diff
@@ -763,7 +763,10 @@ NICs without native support can still be used in
 mode through emulation. Performance is inferior to native netmap
 mode but still significantly higher than sockets, and approaching
 that of in-kernel solutions such as Linux's
-.Xr pktgen .
+.Xr pktgen . Programs using taps such as tcpdump will see packets before they
+are diverted by netmap when emulation is in use. When such packets are
+forwarded to the host stack they show up on the tap a second time, but the host
+OS stack will only see such forwarded packets once.
```

Comment:
We should say "could see" rather than "will see". I agree that you can stress that this is unwanted, just an artifact of the implementation.

We should definitely remove the sentence "When such pakcets are forwarded".. because it's confusing, and it is not something related to the emulated mode. Packets are forwarded by your application (e.g. bridge), this is not part of the generic/emulated. Moreover, the fact that you see packets with TCP dump (the second time, let's say) it's due to the normal functioning of the host RX ring, which is already documented somewhere else.